transport.go

```diff
@@ -22,7 +21,11 @@ const RaftProtocol protocol.ID = "/raft/1.0.0/rpc"
 // which intercepts messages and rewrites them to our own logger
 type logForwarder struct{}
 
-var logLogger = log.New(&logForwarder{}, "", 0)
+var logLogger = hclog.New(&hclog.LoggerOptions{
+	Name:   "raft-net",
+	Output: new(logForwarder),
+	Level:  hclog.DefaultLevel,
+})
```

Comment:
I think this probably breaks the logging since it will not log debug and I have no idea if the log forwarder will match the lines correctly.

There is an `hcLogToLogger` type [here](https://github.com/ipfs-cluster/ipfs-cluster/blob/71bda2d6580b155ee1916d231d50aaffbb7c973d/consensus/raft/logging.go#L23-L138) that could be copy pasted to this library (and made public), and which should do a better job that this horrible string matching I did in the past.

Reply:
thanks for the advice,I will fix this lately